app/components/illustrations/alert.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import * as React from 'react';
+import Svg, {Path} from 'react-native-svg';
+
+function AlertSvgComponent() {
+    return (
+        <Svg
+            width={55}
+            height={55}
+            viewBox='0 0 55 55'
+            fill='none'
+        >
+            <Path
+                d='M4.43715 54.2949C1.46653 54.2949 0.12055 52.1536 1.44612 49.5365L25.4558 2.37353C26.8154 -0.236815 28.9566 -0.236815 30.289 2.37353L54.292 49.5365C55.6515 52.1468 54.292 54.2949 51.3009 54.2949H4.43715Z'
+                fill='#FFBC1F'
```

Comment:
Are this illustrations theme agnostic?

Reply:
The alert illustration shouldn't need any theme colors applied